### DISCUSSION TITLE
希望大佬加入音乐评论功能和支持安卓平板与手机 尤其是平板 太缺HD版本干净的云音乐了 都没人做

### DISCUSSION THREAD

**Discussion by mouzi3102** [qier222/YesPlayMusic#1246] [Q&A]

**Comment by ABucket** [comment]
![image](https://user-images.githubusercontent.com/82869104/149960501-9e9fd06c-57b0-41f2-ab43-6aa7438c8a33.png)
![image](https://user-images.githubusercontent.com/82869104/149960519-1de89ea1-146b-4869-bce3-a3fba664d917.png)
![image](https://user-images.githubusercontent.com/82869104/149960608-24acf3dc-e972-413b-9850-4702ca52709b.png)

**Comment by pan93412** [comment]
後者的話你可以用網頁版，應該夠用了：https://music.qier222.com
前者的話⋯⋯這就是 YesPlayMusic 的賣點之一啊 😂
